### DISCUSSION TITLE
argocd cluster add with AKS cluster

### DISCUSSION THREAD

**Discussion by betorvs** [argoproj/argo-cd#9507] [Q&A] [answered]
Hi all,

I cannot find anything related to it on google and here. 

I just add a new AKS cluster to argo, and it reported as `successful` on settings/clusters:
```
new-us-aks-cluster https://AKS-LONG-URL:443 1.22  Successful
```

But after adding a new application in my git repository, I cannot find it on argocd web interface and I received this error:
```
Unable to load data: applications.argoproj.io "sloth-us" not found
```

I added argocd CRDs to new cluster and when I get argo applications its returns sync and health status as empty:
```
kubectl --context new-us-aks-cluster get applications -n argocd sloth-us
NAME       SYNC STATUS   HEALTH STATUS
sloth-us
```

I tested the connection from argocd pods from my current cluster (on AKS too) to new cluster and it's working.

Any idea about it? I'm a little lost here.

**Comment by betorvs** [comment, marked as answer]
I did a misconfiguration and created argo application in the remote cluster. When I changed it to create on argocd cluster, it started syncing with remote cluster.

**Reply by kgopi1** [reply]
@betorvs , Are you using AKS private cluster ? i cant add my  external AKS private cluster to argocd instance (which runs on another AKS private cluster) . Though VNET peering is in between two clusters.

**Reply by betorvs** [reply]
Hi @kgopi1 ! Yes. I was. Try to debug if your cluster A can connect to azure kubernetes API. Because when you use argocd command line it will get all values from your kubectl config. I hope it helps.
